### ISSUE TITLE
Add example data files

### ISSUE THREAD

**Issue by cdeil** [gammapy/gamma-astro-data-formats#18]
We should add an example data file for every format.

I propose we use run 23037 for HESS with the HAP-HD exporters and std config.
An alternative would be that we add a function that generates a file with the appropriate format and simulated values using Gammapy or Gammalib or just using Python scripts from scratch.

Thoughts?

**Comment by cdeil** [comment]
@mimayer @joleroi @peger089 - Does one of you have thoughts on how this should be done and / or time to work on this in the coming 3 days?

If no, I propose we postpone the example data files to the 0.2 milestone.

**Comment by mimayer** [comment]
We could e.g. create the IRFs and Event lists just using a random generator for each number in the exporters. Would be an issue of 20 minutes I guess.

**Comment by cdeil** [comment]
Yes, writing Python scripts to generate the files is a good option.
Decouples writing of specs from HESS exporter progress and privacy issues ... much better.

We have one example already that @JouvinLea  and I wrote:
https://github.com/gammapy/gamma-astro-data-formats/blob/master/source/irfs/psf/psf_table/make-test-file.py

Random numbers aren't useful. Either fill with zeroes or with useful toy shapes (constant, polynom, power-law, gauss). Filling with toy shapes is more work, but is more useful, because then these files can be used for testing, e.g. a common issue is "did we get the axis order right" or "did we get the binning right (or are 1 or 1/2 bin offset)"?

**Comment by cdeil** [comment]
I'll leave the issue in the 0.1 milestone for now ... if anyone has time to contribute here, please go ahead ... example files should be small (preferably < 10kB, certainly < 100 kB) so that we don't bloat the repo.

If possible, just use numpy, scipy, astropy in the scripts, and not Gammapy / Gammalib, so that it's more stable and independent.

**Comment by joleroi** [comment]
I added an script creating a minimal example file for AEFF2D. For now I reflects what is written on 
http://gamma-astro-data-formats.readthedocs.org/en/latest/irfs/effective_area/index.html
NOT was is actually present in the files written by the HAP exporter (I just discovered that e.g. ALT, ZEN and TELPAT is written there - does this make sense? I mean the info is already in the event list)

**Comment by cdeil** [comment]
@joleroi - Thanks!

Yes, the example files should follow the spec written here, not what the exporters currently do.

It's always OK to store extra header keywords / table columns. This makes the formats extensible and users can adapt them for their use-cases. So it's not a problem if the HAP exporter stores extra keywords. If it's useless or bad to duplicate that info, we can change it though ...

**Comment by cdeil** [comment]
I've added a task list of the example files we need above, so that we can keep track of this task.
